src/src.builds

```diff
@@ -2,9 +2,6 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <ProjectExclusions Condition="'$(OS)' != 'Windows_NT'" Include="$(MSBuildThisFileDirectory)**\src\System.Memory.csproj" />
```

Comment:
Any specific reason we want to do it this way? Having the `ProjectExclusions` here makes it "feel" more temporary than putting it into the Configurations.props file, but maybe that's just me. Putting it into the Configurations.props file makes it feel like the project output is actually Windows-specific.